src/app/create-account/create-account-modal/create-account-modal.component.html

```diff
@@ -0,0 +1,86 @@
+<div class="container"></div>
+<div class="col-lg-6 col-md-6 col-sm-6 col-xs-6 offset-3 float-md-center">
+  <div class="jumbotron" id="create-account-modal">
+    <form action="action_page.php" method="post">
+      <div class="container">
+        <label for="firstname"><b>First Name</b></label>
+        <input
+          type="text"
+          id="fname"
+          placeholder="Enter First Name"
+          name="firstname"
+          required
+        />
+        <div class="container"></div>
+
+        <label for="lastname"><b>Last Name</b></label>
+        <input
+          type="text"
+          id="lname"
+          placeholder="Enter Last Name"
+          name="lastname"
+          required
+        />
+        <div class="container"></div>
+
+        <label for="em"><b>Email</b></label>
+        <input
+          type="email"
+          id="email"
+          placeholder="Enter Email"
+          name="em"
+          size="57"
+          required
+        />
+        <div class="container"></div>
+
+        <label for="address"><b>Address</b></label>
+        <input
+          type="text"
+          id="address"
+          placeholder="Enter Address"
+          name="address"
+          required
+        />
+        <div class="container"></div>
+
+        <label for="address"><b>Phone Number</b></label>
+        <input
+          type="tel"
+          id="phone"
+          placeholder="Enter Phone Number"
+          name="phone"
+          size="57"
+          required
+        />
+        <div class="container"></div>
+
+        <label for="uname"><b>Username</b></label>
+        <input
+          type="text"
+          class="text-line"
+          id="user"
+          placeholder="Enter Username"
+          name="uname"
+          required
+        />
+        <div class="container"></div>
+
+        <label for="psw"><b>Password</b></label>
+        <input
+          type="password"
+          id="pass"
+          placeholder="Enter Password"
+          name="psw"
+          required
+        />
+      </div>
+
+      <div class="container"></div>
+      <button type="submit">Create Account</button>
+      <label>
+        <input type="checkbox" checked="checked" name="remember" /> Remember me
+      </label>
+    </form>
+  </div>
+</div>
```

Comment:
Here's a small form example from my example:
```
        <form [formGroup]="denyFormGroup" (ngSubmit)="denyApplicant(i)">
          <div class="form-group">
            <label for="denialReasonTextArea"
              >Reason for denial of application.</label
            >
            <textarea
              class="form-control"
              id="denialReasonTextArea"
              rows="2"
              formControlName="reason"
              required
            ></textarea>
          </div>
          <div class="form-check">
            <input
              class="form-check-input"
              type="checkbox"
              value=""
              id="sendCopyCheck"
              formControlName="sendCopy"
            />
            <label class="form-check-label" for="sendCopyCheck">
              Send a copy of this response to the applicant's provided email
            </label>
          </div>
          <div class="form-check">
            <input
              class="form-check-input"
              type="checkbox"
              value=""
              id="blacklistCheck"
              formControlName="ban"
            />
            <label class="form-check-label" for="blacklistCheck">
              Blacklist user to prevent future applications
            </label>
          </div>
          <button class="btn btn-secondary float-right ml-3">
            Deny Applicant
          </button>
        </form>
```